source/direct_debit/index.html.md.erb

```diff
@@ -5,248 +5,126 @@ weight: 80
-    <th nowrap>New state</th>
-    <th>Meaning</th>
-    </tr>
-    <tr>
-    <td nowrap><code> CREATED </code></td>
-    <td nowrap><code> STARTED </code></td>
-    <td> Payer has visited the `next_url` and landed on the GOV.UK Pay page to enter their account details. </td>
-    </tr>
-    <tr>
-    <td nowrap><code> STARTED </code></td>
-    <td nowrap><code> PENDING </code></td>
-    <td> Payer has provided their account details, and confirms their consent to the agreement. <code>PENDING</code> is only used with delayed payment types such as Direct Debit. This indicates that payer consent has been given but the agreement has not yet been confirmed with the issuing bank. </td>
-    </tr>
-    <tr>
-    <td nowrap><code> STARTED </code></td>
-    <td nowrap><code> CANCELLED - USER NOT ELIGIBLE </code></td>
-    <td> Payer has indicated that they are not the sole signatory for their bank account, and has selected the link to go back and try a different payment method. </td>
-    </tr>
-    <tr>
-    <td nowrap><code> PENDING </code></td>
-    <td nowrap><code> ACTIVE </code></td>
-    <td> The issuing bank has confirmed that funding source (for example, Direct Debit mandate) is valid. </td>
-    </tr>
-    <tr>
-    <td nowrap><code> PENDING </code></td>
-    <td nowrap><code> FAILED </code></td>
-    <td> The issuing bank has rejected the funding source. Possible reasons: incorrect bank account details, bank account requires multiple signatories to set up a Direct Debit. </td>
-    </tr>
-    <tr>
-    <td nowrap><code> PENDING </code></td>
-    <td nowrap><code> CANCELLED </code></td>
-    <td> Cancelled by the payer. </td>
-    </tr>
-    <tr>
-    <td nowrap><code> ACTIVE </code></td>
-    <td nowrap><code> CANCELLED </code></td>
-    <td> Cancelled by the payer. </td>
-    </tr>
-  </table>
-</div>
+1. Select **Link GoCardless Merchant Account** and follow the on-screen instructions.
 
-## One-off payments using Direct Debit
+You must [contact us](/support_contact_and_more_information/#contact-us) to sign your GoCardless contract before you switch to live.
 
-To take a single payment using Direct Debit, you must set up a Direct Debit mandate.
+## Set up a mandate
 
-1. Create an API key linked to your Direct Debit service.
+You need to set up a Direct Debit 'mandate' before you can take payments. A mandate is your user's agreement that you can take payments from their bank account.
 
-1. Follow the instructions in the [“Payment flow” section](/payment_flow) of this documentation.
+These steps happen in order.
 
-The same API call is used for single payments in all contexts, including
-Direct Debit:
+### 1. GOV.UK Pay creates the mandate
 
-POST /v1/payments
+API call: `POST /v1/directdebit/mandates`
 
+Include the following parameters in your API call:
 
-You cannot take further payments from users using this method without creating
-a new payment. This requires users to re-enter their account details.
+- `description`
```

Comment:
description can be optional

Reply:
Added in https://github.com/alphagov/pay-tech-docs/pull/298/commits/562c05d6cc099a67190a4bfe298a6acc6291794c.